### ISSUE TITLE
Color contrast failing when Multiselect open

### ISSUE THREAD

**Issue by maspychaj-sage** [Sage/carbon#4451]
### Current behaviour

If Multiselect cover an element then it's failing axe color contrast check.
![image](https://user-images.githubusercontent.com/75613789/135818960-6f0b912e-b33b-432b-bfa5-dda2b0e990b5.png)

### Expected behaviour

Covered elements should not create axe errors.

### CodeSandbox or Storybook URL

https://codesandbox.io/s/festive-panka-oi2mw?file=/src/index.js

### Carbon Version

81.0.1

### What browsers are you seeing the problem on?

Chrome

### What Operating System are you seeing the problem on?

MacOS, Windows

### Confidentiality

- [X] I confirm there is no confidential or commercially sensitive information included.

**Comment by nicktitchmarsh** [comment]
@harpalsingh we need UX input on how we can correct these colours please

**Comment by ClaireF-Sage** [comment]
RIV-526

**Comment by clairedenning** [comment]
Why is a DS review needed on this?

**Comment by harpalsingh** [comment]
@clairedenning I asked the same question, trying to understand why the failure is being picked up by the Axe devtools. So it might be a tool issue.

**Comment by DlgSHi** [comment]
Hi @maspychaj-sage, thank you for raising this issue but we quite sure that this is a tool issue instead of the MultiSelect. Closing this one ;)

**Comment by sianford** [comment]
This is not an issue with Carbon nor axe. May be something can be done using the `aria-hidden` attribute to ignore an axe audit on the overlapped component under the Multi Select so it doesn't trigger a violation?
